### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alvinwalker314**
Initial Commit

### CODE REVIEW

**Review by hughjafro — COMMENTED** [review]
### Module 01 Log-Linear:

**Overall:** Nice methodical setup of your data.
**Feedback:** Try running a distplot and barh chart for feature importance..

_Do train/validate/test split._
- ✅ 

_Select at least one additional feature._
- Using a function for data wrangling is always an excellent idea.
  - You could have added your tube_merge here also
    - and use .fillna(value='Missing', axis=1) to address nan
    - then merget on param how='left'

_Fit a model_
- Setting %%time can help to determine how long it takes to run n_estimators=100 and whether it may be worth it to increase/decrease that number.

_Get you validation RMSLE or RMSE with log-transformed targets._
- Having a decent RMSE score after only a few hours is very good.
- What are some ideas that you can execute to get a better score?

_Submit to Kaggle_
- ✅

**Review by hughjafro — COMMENTED** [review]
### Module 02 More Gradient Boosting

**Overall:** Thanks for using comments.
**Feedback:**  Try more hyperparameter tuning.

_Do more feature engineering._
- "Talking" through your code with comments is helpful to those reading your code and also your future self.
- Try distplots 📊 
  - y_train_log = np.log1p(y_train)
sns.distplot(y_train_log);
- Good use of a function to clean/wrangle data
- Good construction of merged components dataframe
- May want to add the %%time magic function to your code to benchmark your model runtime 🏃 
- After you get your results, comment what those scores mean.

_Use XGBoost for gradient boosting._
- Good job with XGBRegressor and adding random_state 🚀
- Try to do more hyperparameter tuning and see if you can get a better score
- What are your thoughts on XGBoost?

_Submit new predictions._
- What is the comparison between public and private score❓

_Commit your notebook to your fork of the GitHub repo._
- ✅
